lib/l10n/app_it.arb

```diff
@@ -299,7 +299,7 @@
   "song_link": "Link della Canzone",
   "skip_this_nonsense": "Salta questa sciocchezza",
   "freedom_of_music": "“Libertà della Musica”",
-  "freedom_of_music_palm": "“Libertà della Musica nel palmo della tua mano”",
+  "freedom_of_music_palm": "“Libertà della Musica sul palmo della tua mano”",
```

Comment:
I'm not sure that "sul palmo" is the right way to use this sentence... Have you any reference?

Reply:
We use "sul" to indicate something that is on something else, while we use "nel" to indicate something that is into something else. In this case, music is on a palm (I mean, you can't put something into a palm).

Reply:
I'm italian, too :)
But I think that in this case the sense of the sentence is figurative. So "nel palmo" is like "nelle tue mani" (in your hands). Not literally on the top of yuor palm. I don't think in italian there is a metaphorical situation where we use "sul tuo palmo".

Reply:
Maybe we could actually put "nelle tue mani"

Reply:
Yeah. I think sound better. Are you agree?

Reply:
Yes. Will update that too